src/main/java/Duke.java

```diff
@@ -52,6 +53,22 @@ public static void main(String[] args) {
                         count2++;
                     }
                     System.out.println("    ____________________________________________________________");
+
+                }else if (userInput.contains("delete")){
+                    content2 = userInput.substring(7);
+                    count4 = Integer.parseInt(content2) - 1;
+                    if(count4 == count){
+                        count = count - 1;
+
+
+                    } else {
+                        while (count4 <= count) {
+                            tasks[count4] = tasks[count4 + 1];
+                            count4++;
+                        }
+                        count = count - 1;
+                    }
+
                 } else {
                     if (userInput.contains("todo")) {
                         content2 = userInput.substring(5);
```

Comment:
You can abstract out this line before all the if-else statements since it is repeated in all 3. Again you can use userInput.substring(userInput.indexOf(" "))